README.md

```diff
@@ -26,8 +26,10 @@ This repository is a [monorepo](https://en.wikipedia.org/wiki/Monorepo) that hol
 
 See [rush.json](./rush.json) for the complete list of packages and [Versioning.md](./Versioning.md) for package and API versioning policies.
 
-Each package has its own **node_modules** directory that contains symbolic links to *common* dependencies managed by Rush.
+## Quick Start
 
+- This is a [sample](https://www.itwinjs.org/sandboxes/iTwinPlatform/3d%20Viewer) of an iTwin viewer - a frontend application that displays infrastructure projects on browsers. It uses many of the APIs and libraries published from this repository.
+- You can also look at [other samples](https://developer.bentley.com/samples/) which showcases the capabilities of iTwin.js, and the iTwin Platform.
 ## Prerequisites
```

Comment:
thoughts on collapsible blocks?

https://gist.github.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab

Reply:
This looks pretty neat. I made a change to move the source code edit workflow section in [CONTRIBUTING.md](https://github.com/iTwin/itwinjs-core/blob/nam/reorg-contribute-doc/CONTRIBUTING.md) to use a collapsible block. The prereqs section of the root README isn't too text heavy yet, so I'm thinking we leave that alone for now, and revisit if the README gets too big